src/CodeGenerator/src/Generator/CodeGenerator/PopulatorGenerator.php

```diff
@@ -56,7 +56,7 @@ class PopulatorGenerator
      */
     private $parserProvider;
 
-    public function __construct(ClassRegistry $classRegistry, NamespaceRegistry $namespaceRegistry, RequirementsRegistry $requirementsRegistry, ObjectGenerator $objectGenerator, ?TypeGenerator $typeGenerator = null, ?EnumGenerator $enumGenerator = null, ?ParserProvider $parserProvider = null)
+    public function __construct(ClassRegistry $classRegistry, NamespaceRegistry $namespaceRegistry, RequirementsRegistry $requirementsRegistry, ObjectGenerator $objectGenerator, TypeGenerator $typeGenerator = null, EnumGenerator $enumGenerator = null, ParserProvider $parserProvider = null)
```

Comment:
I'd kind rather we always enforce that the type should be nullable. I d'no why symfony has gone with this choice. Seems kinda wrong IMO.

Reply:
let's make it explicit